### ISSUE TITLE
Search page improvements

### ISSUE THREAD

**Issue by longhotsummer** [laws-africa/peachjam#403]
* [ ] result must include court for judgments - #429 
* [ ] #493
* [x] don't include facets that don't have values (blocked by #361)
* [x] #405

Notes:
* https://www.loom.com/share/1493f76bff0c48d3b4725bbffecc2a7e
* https://www.loom.com/share/db4fb3e099e34b1399fc147fa20c5a87

**Comment by longhotsummer** [comment]
Closing in favour of individual issues.
